src/openalea/phenomenal/data/data.py

```diff
@@ -38,99 +37,93 @@ def _path_images(plant_number=1, dtype="bin"):
     d : dict of dict of string
         dict[id_camera][angle] = filename
     """
-    data_directory = pkg_resources.resource_filename(
-        'openalea.phenomenal', 'data/plant_{}/{}/'.format(
-            plant_number, dtype))
+    data_directory = os.path.join(name_dir, '{}/'.format(dtype))
 
     d = collections.defaultdict(dict)
     for id_camera in ["side", "top"]:
-        filenames = glob.glob(os.path.join(data_directory, id_camera, '*.png'))
+        filenames = glob.glob(os.path.join(data_directory, id_camera, '*'))
         for filename in filenames:
-            angle = int(os.path.basename(filename).split('.png')[0])
+            angle = int(os.path.basename(filename)[:-4])
```

Comment:
You can get the extension (.jpeg is 5 char)

Reply:
done with last commit